I tried the update, 1.5.4 gives a failure in testing. I didn't get that in 1.5.3 I'm going to poke around a bit more to see what's up.

`nix build .\#nixosTests.tandoor-recipes `

```
cookbook/tests/other/test_recipe_full_text_search.py::test_search_count[found_recipe1-timescooked]
  /nix/store/gy40p8nfs0dnjpbj6c01jk9rks2sc8i9-python3.10-Django-4.2.3/lib/python3.10/site-packages/django/db/models/fields/__init__.py:1535: RuntimeWarning: DateTimeField Recipe.created_at received a naive datetime (2018-09-09 00:00:00) while time zon>
    warnings.warn(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED cookbook/tests/other/test_recipe_full_text_search.py::test_search_date[found_recipe0-viewedon-result0] - assert 0 == 1
FAILED cookbook/tests/other/test_recipe_full_text_search.py::test_search_date[found_recipe1-cookedon-result1] - assert 0 == 1
FAILED cookbook/tests/other/test_recipe_full_text_search.py::test_search_date[found_recipe2-createdon-result2] - assert 11 == 12
FAILED cookbook/tests/other/test_recipe_full_text_search.py::test_search_date[found_recipe3-updatedon-result3] - assert 11 == 12
===== 4 failed, 738 passed, 24 skipped, 639 warnings in 181.12s (0:03:01) ======
```

--- a/pkgs/applications/misc/tandoor-recipes/common.nix
+++ b/pkgs/applications/misc/tandoor-recipes/common.nix
@@ -1,15 +1,15 @@
 { lib, fetchFromGitHub }:
 rec {
-  version = "1.4.12";
+  version = "1.5.4";
 
   src = fetchFromGitHub {
     owner = "TandoorRecipes";
     repo = "recipes";
     rev = version;
-    sha256 = "sha256-ZGPXcpicDYCE+J9mC2Dk/Ds2NYfUETuKXqHxpAGH86w=";
+    sha256 = "sha256-cVrgmRDzuLzl2+4UcrLRdrP6ZFWMkavu9OEogNas2fA=";
   };
 
-  yarnSha256 = "sha256-LJ0uL66tcK6zL8Mkd2UB8dHsslMTtf8wQmgbZdvOT6s=";
+  yarnSha256 = "sha256-0u9P/OsoThP8gonrzcnO5zhIboWMI1mTsXHlbt7l9oE=";
 
   meta = with lib; {
     homepage = "https://tandoor.dev/";

